### PR TITLE
new ioctl command for account create/delete and unit test

### PR DESCRIPTION
**Pull request by RuneAkko**
#1778

### CODE REVIEW

File: ioctl/newcmd/account/accountdelete.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+			if err != nil {
+				return output.NewError(output.AddressError, failToGetAddress, err)
+			}
+			account, err := address.FromString(addr)
+			if err != nil {
+				return output.NewError(output.ConvertError, fmt.Sprintf(failToConvertStringIntoAddress),
+					nil)
+			}
+			ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+				keystore.StandardScryptN, keystore.StandardScryptP)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountdelete.go]: client should contain a keystore

File: ioctl/client.go [hunk 2]
```diff
@@ -38,14 +42,17 @@ type (
...
+		// doing
+		NewKeyStore(string, int, int) *keystore.KeyStore
 	}
 
 	// APIServiceConfig defines a config of APIServiceClient
 	APIServiceConfig struct {
 		Endpoint string
 		Insecure bool
 	}
-
```
Comment (CoderZhi) [inline ioctl/client.go]: revert

File: ioctl/client.go [hunk 3]
```diff
@@ -131,3 +138,30 @@ func (c *client) APIServiceClient(cfg APIServiceConfig) (iotexapi.APIServiceClie
...
+	addr, err := config.GetAddressOrAlias(in)
+	if err != nil {
+		return "", output.NewError(output.AddressError, "", err)
+	}
+	return address(addr)
+}
+
+func (c *client) NewKeyStore(keydir string, scryptN, scryptP int) *keystore.KeyStore {
+	return keystore.NewKeyStore(keydir,
+		scryptN, scryptP)
```
Comment (CoderZhi) [inline ioctl/client.go]: in one line

File: ioctl/newcmd/account/accountcreate.go [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+)
+
+// NewAccountCreate represents the account create command
+func NewAccountCreate(c ioctl.Client) *cobra.Command {
+	var numAccounts uint
+	use, _ := c.SelectTranslation(createUses)
+	short, _ := c.SelectTranslation(createShorts)
+	usage, _ := c.SelectTranslation(createFlagUsages)
+	failToGenerateNewPrivateKey, _ := c.SelectTranslation(failToGenerateNewPrivateKey)
+	failToConvertPublicKeyIntoAddress, _ := c.SelectTranslation(failToConvertPublicKeyIntoAddress)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountcreate.go]: assert on errors

File: ioctl/newcmd/account/accountdelete.go [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+	use, _ := c.SelectTranslation(deleteUses)
+	short, _ := c.SelectTranslation(deleteShorts)
+	failToGetAddress, _ := c.SelectTranslation(failToGetAddress)
+	failToConvertStringIntoAddress, _ := c.SelectTranslation(failToConvertStringIntoAddress)
+	infoWarn, _ := c.SelectTranslation(infoWarn)
+	failToRemoveKeystoreFile, _ := c.SelectTranslation(failToRemoveKeystoreFile)
+	failToWriteToConfigFile, _ := c.SelectTranslation(failToWriteToConfigFile)
+	resultSuccess, _ := c.SelectTranslation(resultSuccess)
+	failToFindAccount, _ := c.SelectTranslation(failToFindAccount)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountdelete.go]: dito

File: ioctl/newcmd/account/accountdelete.go [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+			}
+			account, err := address.FromString(addr)
+			if err != nil {
+				return output.NewError(output.ConvertError, fmt.Sprintf(failToConvertStringIntoAddress),
+					nil)
+			}
+			ks := c.NewKeyStore(config.ReadConfig.Wallet,
+				keystore.StandardScryptN, keystore.StandardScryptP)
+			//ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+			//keystore.StandardScryptN, keystore.StandardScryptP)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountdelete.go]: delete

File: ioctl/newcmd/account/accountdelete.go [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+					delete(config.ReadConfig.Aliases, aliases[addr])
+					out, err := yaml.Marshal(&config.ReadConfig)
+					if err != nil {
+						return output.NewError(output.SerializationError, "", err)
+					}
+					if err := ioutil.WriteFile(config.DefaultConfigFile, out, 0600); err != nil {
+						return output.NewError(output.WriteFileError,
+							fmt.Sprintf(failToWriteToConfigFile, config.DefaultConfigFile), err)
+					}
+					output.PrintResult(fmt.Sprintf(resultSuccess, addr))
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountdelete.go]: we shouldn't referring to package "alias", which was defined in https://github.com/iotexproject/iotex-core/tree/master/ioctl/cmd/alias. Eventually, we need to delete everything in ioctl/cmd

File: ioctl/newcmd/account/accountcreate.go (hunk 4 above)
Comment (RuneAkko) [reply ioctl/newcmd/account/accountcreate.go]: > assert on errors

c.SelectTranslation return `string` and `config.Language`, doesn't return error

File: ioctl/newcmd/account/accountdelete.go (hunk 5 above)
Comment (RuneAkko) [reply ioctl/newcmd/account/accountdelete.go]: > dito

?

**Pull request merged**